### PR TITLE
Clarify recommendation regarding Ibuprofen

### PR DESCRIPTION
**Pull request by VltLgr**
French Health Ministry stated on 14-03-2020 that: Ibuprofen and other anti-inflammatory products are less advised to fight Covid-19 infection as they tend to decrease efficiency of body's natural immune system response, making us then more sensitive to other infections.

Sources:
(en) https://www.theguardian.com/world/2020/mar/14/anti-inflammatory-drugs-may-aggravate-coronavirus-infection
(fr - French Health Minister (who is MD) tweet ) https://twitter.com/olivierveran/status/1238776545398923264

### CODE REVIEW

**Comment by jaysonvirissimo** [comment]
[Here's](https://www.youtube.com/watch?v=EuWo5lmWuZI) a video by Dr. John Campbell discussing the use of NSAIDS in the context of Covid-19.

**Comment by pitviper6** [comment]
Article on the BMJ site about this issue:
[(https://doi.org/10.1136/bmj.m1086)] (Published 17 March 2020)

File: _content/en/03-dont/03-be-on-guard-against-false-hope.md [hunk 1]
```diff
@@ -3,4 +3,4 @@
...
+available products like paracetamol can be useful to lessen some symptoms, [ibuprofen and other anti-inflammatory products are less advised as they tend to decrease efficiency of body's natural immune system response](https://www.theguardian.com/world/2020/mar/14/anti-inflammatory-drugs-may-aggravate-coronavirus-infection); however so far, nothing has been proven to do above random noise. Don't believe individuals who say that crystals, essential oils, etc. have helped them; the folks sharing these products may have the best intentions, or they may be predatory. But the idea is the same: many patients spontaneously do get better on their own because most cases are mild and resolve without pharmaceutical intervention. Instead of worrying about remedies, eat good food. Exercise. Meditate. Rest. Don't neglect your mental health and don't use your anxiety as an excuse for poor behavior towards others.
```
[the remark this answers is not in the document]
Comment (jmcmurry) [reply _content/en/03-dont/03-be-on-guard-against-false-hope.md]: Agree

**Comment by mverzilli** [comment]
@steesdale Feel free to close when you dim it appropriate. OP hasn't come back for a while and it'd be better not to have to keep an eye on this if we need to rethink it from scratch.

**Comment by nicolevasilevsky** [comment]
Can anyone fix the conflicts for this?

File: _content/en/03-dont/03-be-on-guard-against-false-hope.md [hunk 2]
```diff
@@ -3,4 +3,4 @@
 **against false hope of "remedies"**
 
 Be good to your body. Expect companies to capitalize on your fear. Don't buy products that claim to protect or heal you at this point. Currently
-available products like ibuprofen and paracetamol can be useful to lessen some symptoms; however so far, nothing has been proven to do above random noise. Don't believe individuals who say that crystals, essential oils, etc. have helped them; the folks sharing these products may have the best intentions, or they may be predatory. But the idea is the same: many patients spontaneously do get better on their own because most cases are mild and resolve without pharmaceutical intervention. Instead of worrying about remedies, eat good food. Exercise. Meditate. Rest. Don't neglect your mental health and don't use your anxiety as an excuse for poor behavior towards others.
```
Comment (ShaynaKorol) [inline _content/en/03-dont/03-be-on-guard-against-false-hope.md:6]: More recent information: https://www.fda.gov/drugs/drug-safety-and-availability/fda-advises-patients-use-non-steroidal-anti-inflammatory-drugs-nsaids-covid-19?utm_campaign=FDA%20advises%20patients%20on%20use%20of%20NSAIDs%20for%20COVID-19&utm_medium=email&utm_source=Eloqua

**Comment by jmcmurry** [comment]
While I really really appreciate the dedication to get this right, I also think this is the sort of guidance that is so new and so volatile I think it would be best to consider out of scope for FTC from here out. I'm going to close this and also remove the underlying text in a new commit.

**Pull request closed without merging**